user/getting-started.md

```diff
@@ -34,9 +34,10 @@ add the `-v` parameter to keep the `/var/lib/bblfshd` directory in a volume:
 $ docker run -d --name bblfshd --privileged -p 9432:9432 -v /var/lib/bblfshd:/var/lib/bblfshd bblfsh/bblfshd
 ```
 
-(On macOS remove the parameter `-v /var/lib/bblfshd:/var/lib/bblfshd` since the
+On macOS, remove the parameter `-v /var/lib/bblfshd:/var/lib/bblfshd` since the
 default case insensitive filesystem could conflict with the internal drivers'
-case sensitive one.)
+case sensitive one. In that case remember that since this directory will then be
+ephemeral, you'll need to reinstall the drivers after a server restart.
```

Comment:
It would be more precise to indicate _whenever a new container is created_, since you can restart the server without destroying the container, keeping the drivers.

Reply:
Restarting or creating the server container would be maybe more precise, since restarting a docker image (docker restart) also restarts it's ephemeral storage.